test/test_browser.py

```diff
@@ -77,26 +77,6 @@ def do_GET(s):
     httpd.handle_request()
 
 
-def also_with_wasmfs(f):
-  assert callable(f)
-
-  @wraps(f)
-  def metafunc(self, wasmfs, *args, **kwargs):
-    if DEBUG:
-      print('parameterize:wasmfs=%d' % wasmfs)
-    if wasmfs:
-      self.set_setting('WASMFS')
-      self.cflags = self.cflags.copy() + ['-DWASMFS']
-      f(self, *args, **kwargs)
-    else:
-      f(self, *args, **kwargs)
-
-  parameterize(metafunc, {'': (False,),
-                          'wasmfs': (True,)})
-
-  return metafunc
```

Comment:
Comparing to the version the code will now use, the new version has this:
```python
    if wasmfs:
      ..
    else:
      self.cflags += ['-DMEMFS']
```
I suppose that define is harmless, as I see no uses of it in C/CPP files. Perhaps we can even remove it.